node/common/transport/amqp/lib/amqp_receiver.js

```diff
@@ -55,7 +56,8 @@ AmqpReceiver.prototype._onAmqpMessage = function (message) {
      * @event module:azure-iot-amqp-base.AmqpReceiver#message
      * @type {AmqpMessage}
      */
-  this.emit('message', message);
+  var msg = new Message(message.body);
+  this.emit('message', msg);
```

Comment:
The Jsdoc comments say the type of this event is AmqpMessage, but now you're emitting an event with type Message.  Why the difference?